app/src/main/java/ch/epfl/sdp/appart/AdActivity.java

```diff
@@ -44,7 +45,13 @@
     @Inject
     LoginService login;
 
+    public static class Intents {
+        public static final String INTENT_PANORAMA_PICTURES = "panoramas_pictures_references";
+        public static final String INTENT_AD_ID = "adId";
+    }
```

Comment:
It is not better to put  these static class inside for example Util folder? (And maybe add the constants for the other intents)

Reply:
I was thinking that to make it clearer it would be great to keep the intent in the source activity that send messages to a child activity. This way, when you want to retrieve the intend key you to `ParentActivity.Intent.<Key>`. If we put all the intents in some class, I'm afraid we will need to have a very strong naming convention to avoid confusing between the fields. Like if two classes have very similar fields, it could be easy to make a mistake. Nevertheless, I think we must discuss this at the meeting since Carlo already did a class to match this feature.